### DISCUSSION TITLE
anti-desync kick

### DISCUSSION THREAD

**Discussion by fforius** [YimMenu/YimMenu#2647] [General]
still got desync-kick, how to prevent that?

**Comment by pierrelasse** [comment]
https://github.com/YimMenu/YimMenu/issues/2617 is related i think. Maybe you can take a look at it.

**Comment by lonelybud** [comment]
one thing i like about desync kick is that no one knows who did that

**Comment by maybegreat48** [comment]
There used to be a way to block desync kicks, but Rockstar (intentionally) patched that for some reason. The only way to block desync now is to find another loophole in the complaint handler, but I'm not sure if one exists
